web/api/db/migrations/20211227063619_init_db/migration.sql

```diff
@@ -30,7 +30,7 @@ CREATE TABLE "UnsubmittedProfile" (
 CREATE TABLE "CachedProfile" (
     "id" INTEGER NOT NULL,
     "cache" JSONB NOT NULL,
-    "CID" TEXT,
+    "CID" TEXT NOT NULL,
```

Comment:
I don't think I made any changes in prisma.schema that would lead to this change... Must have been a bad cache from before?